nautobot/dcim/models/device_components.py

```diff
@@ -144,6 +144,10 @@ class Meta:
     def get_cable_peer(self):
         return self._cable_peer
 
+    @property
+    def parent(self):
+        raise NotImplementedError("CableTermination models must implement parent()")
+
```

Comment:
Is this still needed/relevant after the other parent/parent_interface changes?

Reply:
TODO Remove